### PR TITLE
7259 add collateral

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#7259

*Description of changes:*

This is the first draft of the add collateral components. The following is missing and will be addressed in the next PR:
- validation of fields
- tests
- some UI tweaks to match ux pin

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by thorwolpert — CHANGES_REQUESTED** [review]
This needs to change in the package file 
"vue-test-utils-helpers": "https://github.com/bcgov/vue-test-utils-helpers.git",

**Comment by jordiwes** [comment]
I did update the vue test utils package

**Review by doug-lovett — APPROVED** [review]

**Comment by jordiwes** [comment]
I have added some tests

File: ppr-ui/src/components/collateral/EditCollateral.vue [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    activeIndex: {
+      type: Number,
+      default: -1
+    },
+    invalidSection: {
+      type: Boolean,
+      default: false
+    }
+  },
+  emits: ['addEditVehicle', 'resetEvent'],
```
Comment (kialj876) [inline ppr-ui/src/components/collateral/EditCollateral.vue:148]: is this used at all or is it for making it easier for us to see what emit options there are? I'm wondering if it should get added to all the composition components I made previously

**Review by kialj876 — APPROVED** [review]
Looks good to me, I just had one question for my own interest

File: ppr-ui/src/components/collateral/EditCollateral.vue (hunk 1 above)
Comment (jordiwes) [reply ppr-ui/src/components/collateral/EditCollateral.vue:148]: I think this is necessary for an emit in the composition api? based on my research. So I think it's used

**Pull request merged**